UTM.xcodeproj/project.pbxproj

```diff
@@ -4322,7 +4322,7 @@
 					"@executable_path/../../../../../../../Frameworks",
 				);
 				MACOSX_DEPLOYMENT_TARGET = 11.0;
-				PRODUCT_BUNDLE_IDENTIFIER = "$(PRODUCT_BUNDLE_PREFIX:default=com.utmapp).QEMULauncher";
```

Comment:
As mentioned in my comment to the PR, please drop the changes from this line onward, thanks.

Reply:
Done in https://github.com/utmapp/UTM/pull/3056/commits/952eb1eb555a6da933864771fce4faae0896317d (force pushed to not mess up commit history)